### PR TITLE
Fixed exception i18n

### PR DESCRIPTION
**Pull request by lzap**
Couple of em

### CODE REVIEW

File: app/models/host/discovered.rb [hunk 1]
```diff
@@ -22,10 +22,10 @@ class Host::Discovered < ::Host::Base
   scope :list, lambda { where(:type => "Host::Discovered").includes(:model, :location, :organization) }
 
   def self.importHostAndFacts facts
-    raise(::Foreman::Exception.new("Invalid Facts, must be a Hash")) unless facts.is_a?(Hash)
+    raise(::Foreman::Exception.new(N_("Invalid Facts, must be a Hash"))) unless facts.is_a?(Hash)
```
Comment (domcleal) [inline app/models/host/discovered.rb]: Change to lower case "Facts", same below.

**Comment by domcleal** [comment]
:+1:

**Comment by GregSutcliffe** [comment]
:+1: with the case change

**Comment by GregSutcliffe** [comment]
Merged as 56b329c51b50db3c5dd0253736ea3c391dc22304 tahnks @lzap !

**Pull request closed without merging**